### ISSUE TITLE
Problem saving a signal

### ISSUE THREAD

**Issue by MAR1TO15** [BruceDevices/firmware#595]
When I capture a signal, from a ringer in this case and reproduce the newly captured signal it works, but when I save it and try to play it from the .sub file it doesn't work
I'm using an M5StickC plus 2 and a C1101 with Bruce 1.8

**Comment by samuelstj** [comment]
Hi. I just faced the same issue.

I also tryied to replay an old .sub and it isn't working.

Some times it saves in the RcSwitch protocol but the key is empyt.

I'm using cardputer with M5 RF modules and Bruce 1.8

**Comment by hunt3rbb666NotFound** [comment]
For me not even the replay on t-embed cc1101 is very strange.

**Comment by JerikoJay** [comment]
Yes, I have the same problem on Bruce 1.8 firmware (m5 stickс P 2)

**Comment by eadmaster** [comment]
this could be related to rolling codes , see the last point if this faq
https://github.com/pr3y/Bruce/wiki/FAQ#irrf-transmitterreceiver-not-working

See also https://github.com/pr3y/Bruce/issues/608

**Comment by bmorcelli** [comment]
Can you please test the last Beta Release?

**Comment by bmorcelli** [comment]
Fixed in 1.8.1
